### PR TITLE
Some CI improvements

### PR DESCRIPTION
**Pull request by gabro**
- fix prettier check
- remove travis badge from README
- add publishing of VSIX to the latest release (the idea is that when the release workflow runs it was triggered by a release)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -22,7 +22,8 @@ jobs:
         id: get_version
         run: echo ::set-output name=NEW_VERSION::${GITHUB_REF#/refs/tags/v}
       - run: yarn install
-      - run: yarn version --no-git-tag-version --new-version ${{
+      - run:
```
Comment (olafurpg) [inline .github/workflows/release.yml]: This is needed to fix the interaction with prettier
```suggestion
      - run: |
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -38,3 +39,19 @@ jobs:
...
+        id: get_upload_url
+        run: |
+          URL=curl --silent "https://api.github.com/repos/scalameta/metals-vscode/releases/latest" | jq -r '.upload_url' | cut -d'{' -f 1
+          echo ::set-output name=UPLOAD_URL::$URL
+      - name: Upload VSIX package to release
+        uses: actions/upload-release-asset@v1.0.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: { { steps.get_upload_url.outputs.UPLOAD_URL } }
```
Comment (olafurpg) [inline .github/workflows/release.yml]: Woah, this is cool!

Comment (gabro) [reply .github/workflows/release.yml]: If it works, sure 😅

File: .github/workflows/release.yml (hunk 1 above)
Comment (gabro) [reply .github/workflows/release.yml]: Thanks, fixed

**Comment by gabro** [comment]
@olafurpg what do you think about triggering a release to see whether this works?

**Comment by olafurpg** [comment]
@gabro sounds good! I think we have a few fixes in master that would be good to release as well

**Pull request merged**